configs/bundle_config.yaml

```diff
@@ -107,6 +107,17 @@ databundles:
     disable_by_opt:
       build_cutout: [all]
 
+  # Cutout of Kazakhstan for CI test of pypsa-kz-data repo, only 5 MB
+  bundle_cutouts_test_KZ:
+    countries: [KZ]
+    category: cutouts
+    destination: "cutouts"
+    urls:
+      zenodo: https://sandbox.zenodo.org/record/1238391/files/test_cutouts_KZ.zip?download=1
+    output: [cutouts/asia-2013-era5.nc]
```

Comment:
let's change the output also to `cutout-2013-era5-tutorial.nc` -- this is our new naming standard since a few months.
also, if the data is not there yet, can you put the tutorial network into our sandbox: https://sandbox.zenodo.org/record/1223907 where the other data is located at?

Reply:
I have already uploaded the file to zenodo to the provided repository. I have updated the link.

Reply:
Thanks @yerbol-akhmetov !
Final change :+1: Can you store the kz-data case with the tutorial naming? 
So not `test_cutouts_KZ.zip` but `tutorial_cutouts_KZ.zip`?
Consistency can help to programmatically extract data in future.